src/constants.js

```diff
@@ -22,8 +22,8 @@ export const JAEGER_CLIENT_VERSION_TAG_KEY = 'jaeger.version';
 // TRACER_HOSTNAME_TAG_KEY used to report host name of the process.
 export const TRACER_HOSTNAME_TAG_KEY = 'jaeger.hostname';
 
-// TRACER_CLIENT_ID_TAG_KEY used to report client ID of the process.
-export const TRACER_CLIENT_ID_TAG_KEY = 'client-id';
+// TRACER_CLIENT_UUID_TAG_KEY used to report client UUID of the process.
+export const TRACER_CLIENT_UUID_TAG_KEY = 'client-uuid';
```

Comment:
at this point we just need to settle on something.

Reply:
I prefer to just go with `client-uuid`, it's a misnomer but at least it's consistent everywhere.

Reply:
I don't really mind, but the issue I have is that it's easier to fix a value than an identifier. Here is my suggestion:

```
CLIENT_ID = 'client-uuid'
```

And once we upgrade backend:

```
CLIENT_ID = 'client-id'
```

But if you do this:

```
CLIENT_UUID = 'client-uuid'
```

Then this will break public API:

```
CLIENT_ID = 'client-id'
```